### PR TITLE
New version: GLab.GLab version 1.23.1

### PR DESCRIPTION
**Pull request by viktomas**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `manifests/g/GLab/GLab/1.23.1` is the name of the directory containing the manifest you're submitting.

-----

Apart from updating the package, this PR also changes the origin (ownership) of the manifest

GitLab is now the official maintainer of the `glab` package.

- Archived original repo: https://github.com/profclems/glab
- Announcement issue: https://github.com/profclems/glab/issues/983
- The new home for the project https://gitlab.com/gitlab-org/cli
- Release mentioned in this PR https://gitlab.com/gitlab-org/cli/-/releases/v1.23.0

I've made a similar PR for

- homebrew: https://github.com/Homebrew/homebrew-core/pull/116608
- scoop: https://github.com/ScoopInstaller/Main/pull/4168

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90349)

### CODE REVIEW

File: manifests/g/GLab/GLab/1.23.1/GLab.GLab.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GLab.GLab
+PackageVersion: 1.22.0
```
Comment (russellbanks) [inline manifests/g/GLab/GLab/1.23.1/GLab.GLab.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.23.1
```

**Review by russellbanks — CHANGES_REQUESTED** [review]

**Comment by russellbanks** [comment]
As a side note, please update your clone of winget-pkgs. The YamlCreate you're using is a bit outdated.

File: manifests/g/GLab/GLab/1.23.1/GLab.GLab.locale.en-US.yaml (hunk 1 above)
Comment (viktomas) [reply manifests/g/GLab/GLab/1.23.1/GLab.GLab.locale.en-US.yaml]: 🤦 Thank you @russellbanks

**Comment by viktomas** [comment]
> As a side note, please update your clone of winget-pkgs. The YamlCreate you're using is a bit outdated.

@russellbanks Ok, I updated to the latest `master` but there have been only 4 commits in it since I cloned it half an hour ago.

**Comment by russellbanks** [comment]
> > As a side note, please update your clone of winget-pkgs. The YamlCreate you're using is a bit outdated.
> 
> @russellbanks Ok, I updated to the latest `master` but there have been only 4 commits in it since I cloned it half an hour ago.

The YamlCreate in your fork is the correct version but this PR was created with 2.0.4 so I'm not sure

**Review by russellbanks — APPROVED** [review]

**Pull request merged**